debug/package.json

```diff
@@ -5,7 +5,6 @@
   "private": true,
   "description": "Preact extensions for development",
   "main": "dist/debug.js",
-  "module": "dist/debug.mjs",
```

Comment:
This change can probably be removed when the new approach works.

Reply:
@marvinhagemeister you mean after the switch to `.module.js`?

Reply:
@developit No, I'm not sure if the change here is needed for this PR in particular.